Debian Bug: #991920
pkg-config is not essential to dracut operation, at least on a regular Debian
system. However, demoting pkg-config to Recommends will allow minimal systems
that try to avoid shipping perl to do so.
--- a/debian/control	2020-09-21 09:09:54.594879264 +0000
+++ b/debian/control	2020-09-21 09:10:08.762876309 +0000
@@ -23,8 +23,8 @@
 Package: dracut-core
 Architecture: any
 Multi-Arch: foreign
-Recommends: cryptsetup, dmsetup, dmraid, lvm2, mdadm, console-setup, binutils, systemd, pigz
-Depends: bash (>= 4), cpio, kmod, udev, kpartx, util-linux (>= 2.20), pkg-config, libkmod2, e2fsprogs, ${shlibs:Depends}, ${misc:Depends}
+Recommends: cryptsetup, dmsetup, dmraid, lvm2, mdadm, console-setup, binutils, systemd, pigz, pkg-config
+Depends: bash (>= 4), cpio, kmod, udev, kpartx, util-linux (>= 2.20), libkmod2, e2fsprogs, ${shlibs:Depends}, ${misc:Depends}
 Breaks: dracut-network (<< 043-2), dracut (<< 044+3-2~)
 Replaces: dracut-network (<< 043-2), dracut (<< 044+3-2~)
 Description: dracut is an event driven initramfs infrastructure (core tools)
